Core/NoOpRunner.Core/Shapes/ShapeFactories/ShapeFactory.cs

```diff
@@ -11,9 +11,10 @@ public override BaseShape GetShape(Shape shape, int x, int y)
         {
             switch (shape)
             {
```

Comment:
Implementing both abstractFactory and a simple factory for shapes is pretty redundant. On the other hand, AbstractFactory impl already contains a Factory impl. Leave as is for now, but we need to consult the prof about this.

Reply:
Correct, its only here for required pattern implementation